src/main/java/javax/money/MonetaryOperator.java

```diff
@@ -60,10 +60,6 @@
  * thread-safe when invoked.
  * </p>
  *
- * <p>
```

Comment:
But why it's not implements the UnaryOperator? Maybe this was just missed so we should implement it instead of removing the comment.